ranger/data/mime.types

```diff
@@ -8,24 +8,30 @@
 #  Mimetypes are used for colorschemes and the builtin filetype detection
 #  to execute files with the right program.
 #
+#  You can find the official list of Media Types assigned by IANA here,
+#  https://www.iana.org/assignments/media-types/media-types.xhtml
+#  We deviate from these in certain cases when formats lack an official type
+#  or the type is too generic, application/* for a video format for example.
+#  In such cases we try to adhere to what file(1) returns.
+#
 ###############################################################################
 
-audio/flac              flac
+application/javascript  js
+
 audio/musepack          mpc mpp mp+
 audio/ogg               oga ogg spx opus
 audio/wavpack           wv wvc
 audio/webm              weba
 audio/x-ape             ape
+audio/x-flac            flac
```

Comment:
Do we need this line? The Python `mimetypes` module already returns it for the FLAC files. If it didn't, I'd go with `audio/flac` which `file(1)` returns.

Reply:
I guess we don't need it. But both `mimetypes` and `file(1)` return `audio/x-flac` for me.

Actually, I'd say we err on the side of caution. #1647 wouldn't be necessary either according the same guidelines but a user *did* experience an issue.

Reply:
Seems like NixOS's purity prevents the mimetypes module from loading system-specific files:
```
[nix-shell:~]$ python
Python 2.7.16 (default, Mar  2 2019, 18:34:01) 
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from mimetypes import guess_type
>>> guess_type('E.mkv')
(None, None)
>>> guess_type('E.flac')
(None, None)
>>> guess_type('E.webp')
(None, None)
>>> guess_type('E.webm')
('video/webm', None)
>>> guess_type('E.weba')
(None, None)
>>> 
```

So I think we should add *a* type for all of these, I don't really care whether we go `flac` or `x-flac`.

Reply:
Isn't it a bug that should be fixed on NixOS's side?

Reply:
Not really imo. These aren't official mime types so the OS doesn't really have any business distributing them.